darts/models/forecasting/baselines.py

```diff
@@ -177,18 +178,26 @@ def __init__(
 
         Parameters
         ----------
-        models
+        forecasting_models
             List of forecasting models whose predictions to ensemble
+        retrain_forecasting_models
+            If set to `False`, the `forecasting_models` are not retrained on `series` (only supported if all the
+            `forecasting_models` are pretrained `GlobalForecastingModels`). Default: ``True``.
         show_warnings
             Whether to show warnings related to models covariates support.
         """
         super().__init__(
-            models=models,
+            forecasting_models=forecasting_models,
```

Comment:
this super call has a lot of type hint warnings. E.g. traing_num_samples is supposed to be `int` but we give `None`. Can you fix these?